pantry-finder-api/app/models/agency.rb

```diff
@@ -13,6 +13,7 @@ class Agency < ApplicationRecord
                     dependent: :restrict_with_exception
   has_many :event_dates, through: :events,
                          dependent: :restrict_with_exception
+  has_many :zip_agencies, through: :events
```

Comment:
This could be `has_many :zip_codes, through: :events`. Having zip_agencies is not valuable as it's just a join table so you'll always be doing one more relation afterwords.

Reply:
We are not joining on ZipCodes but the event_service_geographies table which provides the new rules for what geographic areas the zip code supports.